scripts/portstat

```diff
@@ -27,8 +27,8 @@ PORT_RATE = 40
 
 NStats = namedtuple("NStats", "rx_ok, rx_err, rx_drop, rx_ovr, tx_ok,\
                     tx_err, tx_drop, tx_ovr, rx_byt, tx_byt")
-header = ['Iface', 'RX_OK', 'RX_RATE', 'RX_UTIL', 'RX_ERR', 'RX_DRP', 'RX_OVR',
-          'TX_OK', 'TX_RATE', 'TX_UTIL', 'TX_ERR', 'TX_DRP', 'TX_OVR']
+header = ['Iface','STATE', 'RX_OK', 'RX_BRATE', 'Rx_PRATE', 'RX_UTIL', 'RX_ERR', 'RX_DRP', 'RX_OVR',
+          'TX_OK', 'TX_BRATE', 'Tx_PRATE', 'TX_UTIL', 'TX_ERR', 'TX_DRP', 'TX_OVR']
```

Comment:
we are adding three lines, I really like to limit one row to one line.

by default, let's use the only show RX_BPS, then let's add option to show RX_PPS, or both.